report/3.Theory/LanguageSpecification.tex

```diff
@@ -188,6 +221,47 @@ \subsection{Functions}
 For instance, printing; 
 The print function should be able to print all basic types, so that the user does not have to worry about the type of what he is about to print.
 
+\subsection{Arduino functions}
+Since HCL is supposed to be used on the Arduino platform, it is necessary to take the Arduino's \textbf{\textit{setup}} and \textbf{\textit{loop}} functions into account. 
+These are explained in section \ref{arduinoC}.
+In order to follow this convention, a function called \textbf{\textit{loop}}, which takes a function body as a parameter.
+The \textbf{\textit{loop}} function loops consecutively.
```

Comment:
"loops" should probably be changed to "executes"

Reply:
Changed it to iterates. Fixed